tests/winter_django/test_view.py

```diff
@@ -13,3 +16,19 @@ def test_create_django_urls():
 
     assert len(urls) == 1
     assert urls[0].pattern.regex.pattern == expected_url_pattern
+
+
+def test_create_django_urls_from_routes():
+    expected_url_pattern = (
+        '^with-path-parameters/(?P<param1>[^/]+)/(?P<param2>\\d+)/'
+        '(?P<param3>((one)|(two)))/(?P<param4>[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/'
+        '(?P<param5>((1)|(2)))/$'
+    )
+    component = Component.get_by_cls(APIWithPathParameters)
```

Comment:
Please create a separate API class for the test. We have to cover the case when multiple routes uses the same URL but different HTTP methods. Also these routes could be potentially split into two classes - this also should be tested.

Once it's done I believe we can use the new function in create_django_urls.